.github/workflows/dotnet-format.yml

```diff
@@ -0,0 +1,24 @@
+name: dotnet format
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  check-format:
+    runs-on: windows-latest
+
+    steps:
+    - name: check out code
+      uses: actions/checkout@v2
+
+    - name: Setup .NET Core 3.1
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
+
+    - name: Install format tool
+      run: dotnet tool install -g dotnet-format
+
+    - name: dotnet format
```

Comment:
>dotnet format [](start = 12, length = 13)

I suggest friendly names for all the steps, so if the previous step is "Install format tool", then this step should be "Run format tool".